src/tests/system/tests/test_failover.py

```diff
@@ -80,3 +80,30 @@ def test_failover__reactivation_timeout_is_honored(
     assert (
         f"Primary server reactivation timeout set to {expected} seconds" in log
     ), f"'Primary server reactivation timeout set to {expected} seconds' not found in logs!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.IPA)
+def test_failover__connect_second_family(client: Client, ipa: IPA):
```

Comment:
I'd add ipv6 in the name, since "families" can also refer to ipv4 

```
tail_failover_connect_using_ipv6_second_family
```

Reply:
Renamed but it is actually `connect_using_ipv4_second_family` because IPv6 does not work in this scenario.